### ISSUE TITLE
Simplify and align profile property in Image 3

### ISSUE THREAD

**Issue by azaroth42** [IIIF/api#1275]
From #520 and #1013, the `profile` property in the info.json document for the Image API has some complications:
  * It currently requires an order, but isn't modeled as an rdf:List at the RDF level.
  * If we make it a list, then it would be a different predicate compared to other uses of `profile`
  * It requires a mixed datatype value of first a string, and then an object.  This is contrary to the simple as possible principle, and the agreement for Presentation to always use arrays (#1131)
  * The merge of profile URI and additional functionality description in 2.x was for a good semantic reason (the extra profile extends the base profile) and allows for community specific profiles to be given URIs and asserted in the document ... but no one has even flinched in this direction to our knowledge.  It's currently, thus, over-engineered, even if it is implemented.

Design constraints for how to deal with profile and additional functionality in Image 3.0:
  * Consistent with other APIs' usage of `profile` (single URI?)
  * Be round-trippable through RDF [re #1013]
  * Retain the capabilities of 2.1.1

Proposal:
  * `profile` is a single URI, and SHOULD be from the list of iiif URIs.
  * It SHOULD dereference to a description of the functionality
  * Add a new property that holds the extended information, as per 2.x  (rather than going back to 1.x where it was all at the top level)

**Comment by azaroth42** [comment]
Related to #505.

**Comment by jpstroop** [comment]
@tomcrane @jpstroop @azaroth42 in Toronto: In Image 3, move `supports`, `qualities`, and `formats` out of the `profile` list and up to the top level on the info.json document. Rename `supports` to something like `additional_features` (i.e. beyond those enumerated by the profile).

**Comment by azaroth42** [comment]
And noting the requirement for an implementation the exhibits a different behavior in the presence or lack of `property_formerly_known_as_supports`.

**Comment by tomcrane** [comment]
For discussion on call - sample from 2.1 Image spec

http://iiif.io/api/image/2.1/#complete-response

**Comment by tomcrane** [comment]
I'm +1 on the Toronto summary https://github.com/IIIF/api/issues/1275#issuecomment-335894337

**Comment by azaroth42** [comment]
It would thus look something like...

```json
{
  "@context" : "http://iiif.io/api/image/3/context.json",
  "id": "http://example.com/iiif/identifier",
  "type": "ImageService3",
  "protocol": "http://iiif.io/api/image",
  "profile": "level2",
  "height": 5000,
  "width": 3000,
  "sizes": [ ... ],
  "tiles": [ ... ],
  "qualities": ["default", "color", "gray", "bitonal"],
  "formats": ["jpg", "png", "tiff", "webp"],
  "features": ["sizeAboveFull", "arbitraryRotation"],
}
```

**Comment by azaroth42** [comment]
Consensus on 1/31 call:

* We like the new shape -- much easier to understand and process
* We agree that the properties only list extra/bonus/additional/more values beyond the profile level.
* We agree that there should thus be a prefix to qualities (etc) and that all three should have it
* But didn't pick a particular prefix.  Additional suggestions welcome!

**Comment by zimeon** [comment]
Having missed the call I'll add my :+1: to the proposed share. I don't like `more` as a prefix, either `additional` or `extra` seem fine

**Comment by workergnome** [comment]
minor preference for `additionalFoo`, but that's my blasted formality.

**Comment by azaroth42** [comment]
Closed by #1446
